csbmysql/resource_binding_user_test.go

```diff
@@ -100,8 +108,11 @@ func checkUserIsCreated(username, password string) resource.TestCheckFunc {
 		Expect(rows.Next()).To(BeFalse())
 
 		By("Connecting as the binding user")
-
-		userURI := fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?tls=skip-verify", username, password, dbHost, port, database)
+		tlsMode := "skip-verify"
```

Comment:
thoughts: I believe that `tls=true` would work too

Reply:
Unfortunately, you can't set it to `true` when connecting to localhost:
> tls: failed to verify certificate: x509: “localhost” certificate is not standards compliant